### PR TITLE
Autofix Rubocop Layout rules - Batch  4

### PR DESCRIPTION
**Pull request by nschonni**
Forth run a the autofixing the Layout related Rubocop rules.
Most of these have options allowing overriding specific things if they style isn't preffered.

This is the last of the Layout, except for the 2 that were removed from the last batch

### CODE REVIEW

File: lib/ox/document.rb [hunk 1]
```diff
@@ -7,14 +7,14 @@ class Document < Element
     #   - _:version_ [String] version, typically '1.0' or '1.1'
     #   - _:encoding_ [String] encoding for the document, currently included but ignored
     #   - _:standalone_ [String] indicates the document is standalone
-    def initialize(prolog={})
+    def initialize(prolog = {})
```
Comment (ohler55) [inline lib/ox/document.rb:10]: I thought we had this changed in Ox to allow no space for default argument values?

**Comment by ohler55** [comment]
I noticed that arrays are expected to have no spaces next to the brackets yet hash maps expect spaces. That seems inconsistent to me. Can we be consistent in that regard? It doesn't matter to me which one is selected but consistent would be better.

File: lib/ox/document.rb (hunk 1 above)
Comment (nschonni) [reply lib/ox/document.rb:10]: That has on option of `EnforcedStyle: no_space`, is that the preference for https://docs.rubocop.org/rubocop/cops_layout.html#layoutspacearoundequalsinparameterdefault

Comment (nschonni) [reply lib/ox/document.rb]: OK, set the enforced style and reran that rule

**Comment by nschonni** [comment]
I added the Rubocop dev dependency and gave this a rebase after the 2.7 bump included the `sax_reactor` back in.

If there are particular styles you don't like, if you comment on the line, I can try and look up the rule the changed it and if there is an `EnforcedStyle`

File: examples/obj.rb [hunk 2]
```diff
@@ -21,7 +21,7 @@ def to_s
   end
 end
 
-obj = Classy.new(23, ['abc', {x: true}])
+obj = Classy.new(23, ['abc', { x: true }])
```
Comment (ohler55) [inline examples/obj.rb:24]: Perfect example of a mix of spaces after brackets and no space. I prefer consistent behaviour between `[]` and `{}`. Happy to go with either but not a mix.

Comment (ohler55) [reply examples/obj.rb:24]: Maybe tight, no spaces after `[` or `{` or before `]` or `}` is best if there is no overriding consensus.

**Pull request closed without merging**